presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/CatalogName.java

```diff
@@ -11,15 +11,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.prestosql.plugin.hive;
+package io.prestosql.plugin.base;
 
 import static java.util.Objects.requireNonNull;
 
-public class HiveCatalogName
+public final class CatalogName
 {
     private final String catalogName;
 
-    public HiveCatalogName(String catalogName)
+    public CatalogName(String catalogName)
```

Comment:
Speaking of which: Can you please also provide `getCatalogName` that should be used instead of `toString`

Reply:
I thought about that. But i realized having toString() which behaves _differently_ would be very error-prone.
And thus having a dedicated `get` method is redundant.

I will add javadoc to toString to guarantee current behavior.